### DISCUSSION TITLE
Loaded events for all the public pages

### DISCUSSION THREAD

**Discussion by WayneMcH** [AzuraCast/AzuraCast#7255] [Q&A] [answered]
I see elsewhere that a custom event "player-ready" was created to signal when all the dynamic content on the public player page completed loading.  Were similar events created for the completion of loading the podcast list page and podcast episodes page?  I'm able to pick up the player-ready event, but it is content on the podcast pages I'm trying to adjust.
I'm trying to eliminate the link on the podcast author, but even deleting the author still leaves an unwanted "mailto:" link.  I'd rather have the author name but just as text and not a link.  If there is a way to do that through admin, I haven't found it.

**Comment by BusterNeece** [comment]
I'd definitely not do that via JS and instead do it via CSS.

**Reply by WayneMcH** [reply]
I can make it disappear in CSS, but I can't strip the anchor but keep the text - can I?  I know you can do lots of funky things in CSS, but that is a trick I haven't learned yet.  Ideally, I'd like to keep the author, but not as a link.  Wow on the lightning-fast response!

**Reply by WayneMcH** [reply]
I have successfully removed the content related to the source in css, with 
_div#podcast h5 small, div#podcast h4 small, div#podcast h5 br, div#podcast h4 br {
  display: none;
}_

I had success in clearing the link, setting up a timer, selecting on _div#podcast h5 small, div#podcast h4 small_ , and if not null, setting the innerHTML to the innerText.  However the link shows until the timer fires, and timers are ugly solutions.  As it stands I can incorporate whose podcast it is in the description, so hiding the author is ok in this instance.

The now non-urgent question remains, for JS customizing on the podcast pages, is there a loaded-completed event that can be used, as there is on the player page?

**Reply by Vaalyn** [reply, marked as answer]
You can disable a link with CSS only so that it's functioning as a link like this:
```
pointer-events: none;
cursor: default;
text-decoration: none;
color: white;
```

**Reply by WayneMcH** [reply]
Thanks @Vaalyn, that works perfectly! You taught me something this morning.  In all my website scripting I've never come across that one.
